[2017-09-24 21:36:56.056] [ERROR] normal - 请求获取验证码连接异常
[2017-09-24 21:37:01.096] [ERROR] normal - 请求获取验证码连接异常
[2017-09-24 21:39:16.439] [ERROR] normal - 请求获取验证码连接异常
[2017-09-24 21:41:24.783] [ERROR] normal - 请求获取验证码连接异常
[2017-09-24 21:42:43.804] [ERROR] normal - 请求获取验证码连接异常
[2017-09-24 21:42:53.803] [ERROR] normal - 请求获取验证码超时
[2017-09-24 21:43:07.067] [ERROR] normal - 请求获取验证码连接异常
[2017-09-24 21:43:17.068] [ERROR] normal - 请求获取验证码超时
[2017-09-24 22:03:30.943] [INFO] normal - { phone: '18088821980',
  code: 'sdfsdf',
  password: 'b3f821e872c29b660f737b843f0a667e',
  platformID: '7612486',
  refuserid: '50037' }
[2017-09-24 22:03:30.944] [INFO] normal - { sAccount: '18088821980',
  sNickName: '未设置昵称',
  sPassword: 'b3f821e872c29b660f737b843f0a667e',
  iSex: 1,
  sProvince: '',
  sCity: '',
  sSignature: '',
  sEmail: '',
  sHeadImage: '',
  iPhoneAuth: 1,
  lPlatformID: '7612486',
  lPushManId: '50037',
  sCode: 'sdfsdf' }
[2017-09-24 22:03:31.086] [INFO] normal - { iRet: -8, sMsg: '手机短信验证码不存在' }
[2017-09-24 22:03:53.610] [ERROR] normal - 请求获取验证码连接异常
[2017-09-24 22:11:19.970] [ERROR] normal - 请求获取验证码连接异常
[2017-09-24 23:32:01.863] [ERROR] normal - { Error: socket hang up
    at TLSSocket.onHangUp (_tls_wrap.js:1124:19)
    at TLSSocket.g (events.js:292:16)
    at emitNone (events.js:91:20)
    at TLSSocket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9) code: 'ECONNRESET' }
Error: socket hang up
    at TLSSocket.onHangUp (_tls_wrap.js:1124:19)
    at TLSSocket.g (events.js:292:16)
    at emitNone (events.js:91:20)
    at TLSSocket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
[2017-09-24 23:40:28.740] [ERROR] normal - { Error: connect ECONNREFUSED 192.168.168.158:3240
    at Object.exports._errnoException (util.js:1020:11)
    at exports._exceptionWithHostPort (util.js:1043:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1086:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '192.168.168.158',
  port: 3240 }
Error: connect ECONNREFUSED 192.168.168.158:3240
    at Object.exports._errnoException (util.js:1020:11)
    at exports._exceptionWithHostPort (util.js:1043:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1086:14)
